kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/KroxyliciousST.java

```diff
@@ -74,7 +77,7 @@ void produceAndConsumeMessage(String namespace) {
     void restartKafkaBrokers(String namespace) {
         String topicName = "my-topic2";
         String message = "Hello-world";
-        int numberOfMessages = 20;
+        int numberOfMessages = 25;
```

Comment:
What is the significance of 25 vs 20 messages being produced/consumed?

Reply:
It was just a matter of being a bit more time sending messages when restarting a pod, but not really significant